sd_jwt/bin/sd_jwt

```diff
@@ -0,0 +1,317 @@
+#!/usr/bin/env python3
+import argparse
+import json
+import logging
+import random
+import sys
+
+from jwcrypto.jwk import JWK
+from sd_jwt import __version__
+from sd_jwt.operations import (
+    create_sd_jwt_and_svc,
+    create_release_jwt,
+    generate_salt,
+    verify
+)
+from typing import Union
+
+logger = logging.getLogger(__name__)
+
+
+def print_repr(values: Union[str,list], nlines=2):
+    value = '\n'.join(values) if isinstance(values, (list, tuple)) else values
+    _nlines = '\n' * nlines if nlines else ""
+    print(f"{value}{_nlines}")
+
+
+def get_jwk(no_randomness: bool = False):
+    """
+    returns static of random JWK
+    """
+    random.seed(0)
+    if no_randomness:
+        ISSUER_KEY = JWK.from_json(json.dumps(ISSUER_KEY_DICT))
+        HOLDER_KEY = JWK.from_json(json.dumps(HOLDER_KEY_DICT))
+        logger.warning("Using fixed randomness for demo purposes")
+    else:
+        ISSUER_KEY = JWK.generate(key_size=DEFAULT_KEY_SIZE, kty=DEFAULT_KTY)
+        HOLDER_KEY = JWK.generate(key_size=DEFAULT_KEY_SIZE, kty=DEFAULT_KTY)
+    
+    ISSUER_PUBLIC_KEY = JWK.from_json(ISSUER_KEY.export_public())
+
+    return dict(
+        ISSUER_KEY = ISSUER_KEY,
+        HOLDER_KEY = HOLDER_KEY,
+        ISSUER_PUBLIC_KEY = ISSUER_PUBLIC_KEY
+    )
+
+
+parser = argparse.ArgumentParser(
+    description=f"{__file__} demo.",
+    epilog=f"{__file__}",
+    formatter_class=argparse.RawTextHelpFormatter
+)
+parser.add_argument(
+    '-t', '--type', required=False,
+    choices=(
+        'simple', 
+        'structured',
+        'complex',
+    ),
+    default='structured',
+    help="Debug level, see python logging; defaults to INFO if omitted"
+)
+parser.add_argument(
```

Comment:
this is actually not used :)

yes, I can bind it and also we may add some relevant loggin call in the `operations.py` and `walk.py` files